dace/sdfg/sdfg.py

```diff
@@ -324,6 +323,19 @@ def from_json(cls, json_obj, context_info=None):
 
         return ret
 
+    def hash_sdfg(self) -> int:
+        '''
+        Returns an hash of the current SDFG, withou considering IDs and attributes name
+        :return:
+        '''
+        jsondict = self.to_json()  # No more nonstandard objects
```

Comment:
this would still not work with nested SDFGs in nested SDFGs. We need to probably walk into the entire key structure of the dictionary and delete all `sdfg_list_id` and `['attributes']['name']` keys